src/assets.php

```diff
@@ -30,16 +32,14 @@ function action_wp_enqueue_scripts() {
 	|
 	*/
 
-	/* phpcs:ignore Squiz.PHP.CommentedOutCode.Found
-	 * wp_enqueue_script(
-	 *     'create-wordpress-plugin-example-entry',
-	 *     get_entry_asset_url( 'example-entry' ),
-	 *     get_asset_dependency_array( 'example-entry' ),
-	 *     get_asset_version( 'example-entry' ),
-	 *     true
-	 * );
-	 * wp_set_script_translations( 'create-wordpress-plugin-example-entry', 'create-wordpress-plugin' );
-	 */
+	wp_enqueue_script(
+		'create-wordpress-plugin-example-entry',
+		get_entry_asset_url( 'example-entry' ),
+		get_asset_dependency_array( 'example-entry' ),
+		get_asset_version( 'example-entry' ),
+		true
+	);
+	wp_set_script_translations( 'create-wordpress-plugin-example-entry', 'create-wordpress-plugin' );
```

Comment:
Did you mean to uncomment the code?

My fear is that these example scripts will be enqueued and forgotten about which makes me wonder ...
should we have example code as an opt-in when running the Make file?

Reply:
I didn't mean to uncomment this!